.travis.yml

```diff
@@ -28,6 +28,8 @@ matrix:
       env: DEPENDENCIES=dev
     - php: 7.0
     - php: 7.1
+    - php: 7.2
+      env: SYMFONY_PHPUNIT_VERSION=6.5
```

Comment:
no need to specify it. The bridge already handles it in recent versions

Reply:
But maybe this should be fixed in the bridge instead. symfony/symfony#26110